cluster/aws/config-default.sh

```diff
@@ -43,6 +43,7 @@ MINION_SCOPES=""
 POLL_SLEEP_INTERVAL=3
 PORTAL_NET="10.0.0.0/16"
 MASTER_IP_RANGE="${MASTER_IP_RANGE:-10.246.0.0/24}"
+CONTAINERS_CIDR=10.244.0.0/24
```

Comment:
If my understanding is correct `CONTAINERS_CIDR` is meant to reflect the ip range for any of the minions?

If so, shouldn't the CIDR be something like `10.244.0.0/16`? Otherwise, we'd only be seeing local traffic as traffic from the range 10.244.0.0-255 when we really want anything from the possible `MINION_IP_RANGES`.